web/src/app/main/components/ToolbarTitle.js

```diff
@@ -116,12 +120,13 @@ const mapStateToProps = state => {
 @connect(mapStateToProps)
 export default class ToolbarTitle extends React.Component {
   renderTitle = title => {
+    document.title = `Go Alert || ${title}`
```

Comment:
as this is a string, is showing the `||` intentional, or is this supposed to be an `or` expression?

Reply:
This is intentional. I've used it before but can change it to anything or remove it altogether.